bucket/openOCD.json

```diff
@@ -1,10 +1,10 @@
 {
-    "version": "20201228",
+    "version": "20210729",
     "description": "OpenOCD is an open-source tool that allows debugging various ARM devices with GDB using a wide variety of JTAG programmers.",
     "homepage": "https://gnutoolchains.com/arm-eabi/openocd/",
```

Comment:
This is official website right?

https://openocd.org/

Reply:
Thank you! Will fix official website.